config/ocaml_flags.sexp

```diff
@@ -1,3 +1,3 @@
-(-w A-4-31-33-34-39-41-42-43-44-45-48-49-50-58-66
- -safe-string -strict-sequence -strict-formats
+(-w A-4-31-33-34-39-41-42-43-44-45-48-49-50-58-66-70 -g
+ -safe-string -strict-sequence -strict-formats -I +str -I +compiler-lib
```

Comment:
this -I is meh and might not work, add str and compiler-libs dependencies in opam and dune properly